tests/console/suse_module_tools.pm

```diff
@@ -38,7 +38,7 @@ sub run {
     # testing modsign-verify command
     if (is_sle('<15-sp1')) {
         # Run command and grep correct output
-        assert_script_run("modsign-verify $pth | grep 'good signature\\|bad signature\\|certificate not found\\|module not signed\\|other error'");
+        assert_script_run("modsign-verify $pth | grep 'good signature\\|bad signature\\|certificate not found\\|module not signed\\|other error\|Invalid signature format'");
```

Comment:
@BillAnastasiadis this is your suggested solution. Is that all right?

Reply:
Yes, that seems correct. Should fix the problem.

Reply:
Just a comment - I suggest escaping the last \ (other error\\|Invalid signature format'")